UPGRADE.md

```diff
@@ -2,6 +2,26 @@
 
 ## 2.5.0
 
+### JS Dependencies update
+
+Sulu requires now the following new versions JS packages for admin build:
```

Comment:
The JavaScript dependencies of the Sulu administration interface were updated to the following versions: